hummingbot/client/hummingbot_application.py

```diff
@@ -99,6 +99,9 @@ def __init__(self):
         self._script_iterator = None
         self._binance_connector = None
 
+        # gateway variables
+        self._shared_client = None
```

Comment:
This shouldn't be here, it should be a part of Gateway base connector I mentioned above.